RDMkit FAIR Cookbook links (#959)
elixir-europe/rdmkit: pages/contribute/editorial_board_guide.md

### Page ID

To find out what the `page_id` of an RDMkit page is, please check its metadata attribute `page_id` at the top of the markdown file or the [Website overview page](website_overview).

> ### Links between RDMkit and FAIR Cookbook

Reviewer: ```suggestion
### Linking from RDMkit to FAIR Cookbook
```

Since this is the RDMkit editorial process I guess we can say it directly this way?

> - To register links between the RDMkit pages  and FAIR Cookbook recipies enter the links in the faircookbook_rdmkit_mapping.yml file. The .yml is placed in the [repository](https://github.com/elixir-europe/faircookbook-rdmkit).

Reviewer: ```suggestion
- Links between RDMkit and FAIR Cookbook are described in the `faircookbook_rdmkit_mapping.yml` file located in the [faircookbook-rdmkit repository](https://github.com/elixir-europe/faircookbook-rdmkit).
```

> - After adding the links create a PR. The PR should be reviewd and approved by atleast one editor from both teams.

Reviewer: ```suggestion
- After adding the links create a PR. The PR should be reviewed and approved by at least one editor from both teams.
```
